Build/Customizations/SqlServer/LinuxSqlServerDockerImage.cs

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using DotNet.Testcontainers.Networks;
+using JetBrains.Annotations;
+using Serilog;
+
+namespace Customizations.SqlServer;
+
+public class LinuxSqlServerDockerImage : ISqlDockerImage
```

Comment:
there is a lot of duplication between Linux and Windows. should we introduce a baseclass?

Reply:
okay.